client/app/common/translations/Messages_cs_CZ.xml

```diff
@@ -136,6 +136,7 @@
    <translation id="common_ipv6" qtlid="34126">IPv6</translation>
    <translation id="common_bitFormat" qtlid="122107">bitů</translation>
    <translation id="common_expired" qtlid="257941">Služba nebyla obnovena. To je způsobeno Vaším rozhodnutím, nebo opožděním platby na Vaší straně.</translation>
+   <translation id="common_settle_bill">Settle</translation>
```

Comment:
Must be done on our internal tool. [Only `fr_FR.xml`](https://github.com/ovh-ux/ovh-manager-dedicated/blob/master/.github/CONTRIBUTING.md#translations) files can be updated.

Reply:
I can't find `common_settle_bill` in `Messages_**_**.xml`.